runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordPlugin.java

```diff
@@ -381,6 +381,12 @@ private void checkForAreaUpdate()
 			return;
 		}
 
+		if (client.getWorldType().contains(WorldType.DEADMAN))
```

Comment:
Why are you implementing this differently than current PVP world check? Just use the same logic and in same method

Reply:
Actually nvm, that is already contained in PVP check. I don't understand why anything extra needs to be implemented

Reply:
Well if you were okay with just hiding the activity completely, similar to PVP, no changes would be necessary. But in #11430 you had mentioned adding the activity line "Playing Deadman Mode", which is what's implemented here and does require the additional changes. If you just want to hide the activity, feel free to close the PR.

Reply:
Imo it is fine to display playing DMM and I would be happy merging that, my only requirement is for it to be implemented for all cases we are skipping atm (e.g all types of PVP worlds)

Reply:
Gotcha.

Reply:
Resolved with 30eab1a.